src/assets/scripts/client/parsers/aircraftCommandParser/aircraftCommandMap.js

```diff
@@ -54,67 +19,191 @@ const UNICODE_COMMANDS = {
-    d: 'altitude',
-    altitude: 'altitude',
-    a: 'altitude',
-    takeoff: 'takeoff',
-    to: 'takeoff',
-    cto: 'takeoff',
-    fph: 'flyPresentHeading',
-    heading: 'heading',
-    fh: 'heading',
-    h: 'heading',
-    turn: 'heading',
-    t: 'heading',
-    speed: 'speed',
-    slow: 'speed',
-    sp: 'speed',
-    '+': 'speed',
-    '-': 'speed',
-    ils: 'land',
-    i: 'land',
-    land: 'land',
-    l: 'land',
-    '*': 'land',
-    reroute: 'reroute',
-    rr: 'reroute',
-    route: 'route',
-    f: 'fix',
-    fix: 'fix',
-    track: 'fix',
-    direct: 'direct',
-    pd: 'direct',
-    dct: 'direct',
-    hold: 'hold',
-    squawk: 'squawk',
-    sq: 'squawk',
-    delete: 'delete',
-    del: 'delete',
-    kill: 'delete',
-    sa: 'sayAltitude',
-    saa: 'sayAssignedAltitude',
-    sh: 'sayHeading',
-    sah: 'sayAssignedHeading',
-    si: 'sayIndicatedAirspeed',
-    sas: 'sayAssignedSpeed'
+    abort: {
+        aliases: ['abort'],
+        functionName: 'runAbort',
+        isSystemCommand: false
+    },
+    airport: {
+        aliases: ['airport'],
+        functionName: '',
+        isSystemCommand: true
+    },
+    altitude: {
+        aliases: ['a', 'altitude', 'c', 'climb', 'd', 'descend'],
+        functionName: 'runAltitude',
+        isSystemCommand: false
+    },
+    auto: {
+        aliases: ['auto'],
+        functionName: '',
+        isSystemCommand: true
+    },
+    clear: {
+        aliases: ['clear'],
+        functionName: '',
+        isSystemCommand: true
+    },
+    clearedAsFiled: {
+        aliases: ['caf', 'clearedAsFiled'],
+        functionName: 'runClearedAsFiled',
+        isSystemCommand: false
+    },
+    climbViaSid: {
+        aliases: ['climbViaSid', 'cvs'],
+        functionName: 'runClimbViaSID',
+        isSystemCommand: false
+    },
+    delete: {
+        aliases: ['del', 'delete', 'kill'],
+        functionName: 'runDelete',
+        isSystemCommand: false
+    },
+    descendViaStar: {
+        aliases: ['descendViaStar', 'dvs'],
+        functionName: 'runDescendViaStar',
+        isSystemCommand: false
+    },
+    direct: {
+        aliases: ['dct', 'direct', 'pd'],
+        functionName: 'runDirect',
+        isSystemCommand: false
+    },
+    expectArrivalRunway: {
+        aliases: ['e'],
+        functionName: 'runExpectArrivalRunway',
+        isSystemCommand: false
+    },
+    fix: {
+        aliases: ['f', 'fix', 'track'],
+        functionName: 'runFix',
+        isSystemCommand: false
+    },
+    flyPresentHeading: {
+        aliases: ['fph'],
+        functionName: 'runFlyPresentHeading',
+        isSystemCommand: false
+    },
+    heading: {
+        aliases: ['fh', 'h', 'heading', 't', 'turn'],
+        functionName: 'runHeading',
+        isSystemCommand: false
+    },
+    hold: {
+        aliases: ['hold'],
+        functionName: 'runHold',
+        isSystemCommand: false
+    },
+    land: {
+        aliases: ['*', 'i', 'ils'],
```

Comment:
be careful with the `*` again here.  this may not be interpreted the same across operating systems

Reply:
Added #919 to look into it